.github/workflows/json-validator.yaml

```diff
@@ -0,0 +1,31 @@
+name: Validate JSON Files
+
+on:
+  push:
+    paths:
+      - '**/*.json'
+  pull_request:
+    paths:
+      - '**/*.json'
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Validate JSON
+        run: |
+          for file in $(find . -type f -name '*.json'); do
```

Comment:
this should exclude `.venv` from the `find` - might not be a problem in CI but better for running locally

Reply:
oh! good point!